### PR TITLE
Update CLI integration test

### PR DESCRIPTION
**Pull request by bitoku**
### Which issue this PR addresses:

Fixes https://issues.redhat.com/browse/ARO-8293

### What this PR does / why we need it:

This PR implements a scenario-based integration test for all subcommands.
This will increase the test coverage in ARO-RP repo, and will reduce the test execution time in upstream because it doesn't create a cluster for each subcommand.

I'm going to open a PR in upstream after this PR is merged.

### Test plan for issue:

Run `azdev test`.
https://github.com/Azure/ARO-RP/blob/master/docs/az-aro-python-development.md#tests

### Is there any documentation that needs to be updated for this PR?

na

### How do you know this will function as expected in production? 

not production feature.

### CODE REVIEW

File: python/az/aro/azext_aro/tests/latest/integration/test_aro_scenario.py [hunk 1]
```diff
@@ -2,42 +2,93 @@
...
+        temp_kubeconfig_path = self.create_random_name('kubeconfig', 24) + '.tmp'
+
         self.kwargs.update({
-            'name': 'test1'
+            'name': name,
+            'resource_group': resource_group,
+            'subscription': subscription,
+            'master_subnet': master_subnet,
+            'worker_subnet': worker_subnet,
+            'master_ip_range': '10.{}.{}.0/24'.format(randint(0, 127), randint(0, 255)),
```
Comment (tsatam) [inline python/az/aro/azext_aro/tests/latest/integration/test_aro_scenario.py:39]: We don't need these IP ranges to be random if we're making the vnet these are defined on, right?

**Review by tsatam — DISMISSED** [review]
Looks good, there's a couple non-blocking questions/followups that can either be handled here or in follow-up efforts.

File: python/az/aro/azext_aro/tests/latest/integration/test_aro_scenario.py (hunk 1 above)
Comment (bitoku) [reply python/az/aro/azext_aro/tests/latest/integration/test_aro_scenario.py:39]: That's true, but the upstream code is written like this, and I can't find any downside to write in this way, so I'll keep it as it is.
https://github.com/Azure/azure-cli/blob/8de4e1305646c0bc931bd50f29377761c5fd324f/src/azure-cli/azure/cli/command_modules/aro/tests/latest/test_aro_commands.py#L34-L44

**Review by tsatam — APPROVED** [review]

**Pull request merged**